app/controllers/topbar_api_controller.rb

```diff
@@ -0,0 +1,33 @@
+# coding: utf-8
+class TopbarApiController < ApplicationController
+
+  attr_reader :community_customization
```

Comment:
Do I understand this right: ApplicationController has instance variable `@community_customization` and now you're adding a getter method to it in `TopbarApiController`? If yes, then why not add the getter to `ApplicationController`? Or maybe you could at least add a comment here that the instance variable is in `ApplicationController`. It's a bit confusing when there's the `attr_reader` but no instance variable in this class, so adding a comment which explains where the instance variable actually is would help understanding this code.